src/main/java/uk/gov/hmcts/reform/waworkflowapi/controllers/startworkflow/CreateTaskController.java

```diff
@@ -18,7 +18,8 @@
 @RestController
 public class CreateTaskController {
 
-    private final TaskService taskService;
+    @Autowired
+    private TaskService taskService;
```

Comment:
we said we were going to do this with constructor autowiring so should be 

public CreateTaskController(TaskService taskService) {
        this.taskService = taskService;
    }

we don't need the the @Autowired that was on there before.

Reply:
yes, done.